docs/source/Examples/example_3J_with_DA_solver.rst

```diff
@@ -28,7 +27,7 @@ Example of a 3J solar cell calculated with the DA solver
 
 
     def this_dir_file(f):
-        return os.path.join(os.path.split(__file__)[0], f)
+        return "data/" + f
```

Comment:
This is not equivalent to the previous version. It assumes that the present working directory is the containing directory and that the separator character in the path is a forward slash, none necessarily true. 

I you want a (possibly) more elegant, but still robust approach I suggest you use `Path`:

```suggestion
        from pathlib import Path
        return str(Path(__file__).parent / "data" /  f)
```